posix.mak

```diff
@@ -370,6 +370,9 @@ style_lint:
 	@echo "Check for trailing whitespace"
 	$(GREP) -nr '[[:blank:]]$$' $(MANIFEST) ; test $$? -eq 1
 
+	@echo "Enforce whitespace before opening parenthesis"
+	grep -nrE "\<(for|foreach|foreach_reverse|if|while|switch|catch)\(" $$(find src -name '*.d') ; test $$? -eq 1
+
```

Comment:
Can you add a check for ~`version ( abc )`~ -> `version (abc)`

Reply:
I have it in a separate branch, do you really want it all squashed together?  This change alone is big enough as it is.

Reply:
There's loads more styling checks that would be nice to copy from phobos as well.  But I'm not going to spend too much time on that.

Reply:
> do you really want it all squashed together?

No.